docs/development.md

```diff
@@ -0,0 +1,107 @@
+and follow the installation instructions for each application.
+
+Accounts can be added to the `$config` in your git-ignored `simplesamlphp/config/authsources.php` file. 
+For development purposes, it could be useful to add SRAA, RAA and RA accounts as well as users for each
+second factor.
+
+```php
+    ...
+        'sraa:sraa' => array(
+            'commonName'             => 'SRAA',                 // displayName
+            'mail'                    => 'sraa@sraa.example',   // email (also used for EPTI)
+            'schacHomeOrganization'   => 'Example Inc',         // schacHomeOrganisation
+        ),
+        'raa:raa' => array(
+            'commonName'             => 'RAA',                  // displayName
+            'mail'                    => 'raa@raa.example',     // email (also used for EPTI)
+            'schacHomeOrganization'   => 'Example Inc',         // schacHomeOrganisation
+        ),
+        'ra:ra' => array(
+            'commonName'             => 'RA',                   // displayName
+            'mail'                    => 'ra@ra.example',       // email (also used for EPTI)
+            'schacHomeOrganization'   => 'Example Inc',         // schacHomeOrganisation
+        ),
+        'sms:sms' => array(
+            'commonName'            => 'SMS',                   // displayName
+            'mail'                  => 'sms@sms.example',       // email (also used for EPTI)
+            'schacHomeOrganization' => 'Example Inc',           // schacHomeOrganisation
+        ),
+        'yubi:yubi' => array(
+            'commonName'            => 'Yubi',                  // displayName
+            'mail'                  => 'yubi@yubi.example',     // email (also used for EPTI)
+            'schacHomeOrganization' => 'Example Inc',           // schacHomeOrganisation
+        ),
+        'u2f:u2f' => array(
+            'commonName'            => 'U2F',                   // displayName
+            'mail'                  => 'u2f@u2f.example',       // email (also used for EPTI)
+            'schacHomeOrganization' => 'Example Inc',           // schacHomeOrganisation
+        ),
+    ...
+```
+
+To whitelist the organisation of these accounts, use the [Middleware Manager API][middleware-manager]:
+
+```
+    POST /app_dev.php/management/whitelist/add HTTP/1.1
+
+     {
+       "institutions": [
+         "Example Inc"
+       ]
+     }
+```
+
+To allow account `sraa` to be added as an SRAA, use the [Middleware Manager API][middleware-manager] 
+(request body has been truncated) to configure its NameID as such: 
+
+```
+      POST /app_dev.php/management/configuration HTTP/1.1
+
+      {
+          "sraa": [
+            "9971dbcf01267b11f6107d9cafb43e5b4009a955"
+          ],
+          ...
+      }
+```
+
+To add account `sraa` by its NameID as an SRAA, run the following for Stepup-Middleware, replacing `<YOUR_YUBIKEY_PUBLIC_ID` 
+with the number relating to your Yubikey (it is usually printed on it):
```

Comment:
also, ensure proper NameID :wink: (in this case it is `9971...`, but when using a different account configured above, you need to use the corresponding NameID)